README.md

```diff
@@ -15,6 +15,7 @@
 | 9 | [`OceanFloor`, `Basin`][day9] | Part [1][day9-1], [2][day9-2] | [Marking methods with `@discardableResult`][discardable] to avoid compile-time warnings about unused return values
 | 10 | [`ChunkLineParser`][day10] | Part [1][day10-1], [2][day10-2] | [Defining][enum-associated-define], [instantiating][enum-associated-init], and [consuming][enum-associated-consume] enumerations with associated values
 | 11 | [`EnergySimulator`][day11] | Part [1][day11-1], [2][day11-2] | Generating a [changelog][changelog] by running [`npx standard-version`][standard-version]
+| 12 | [`CaveNetwork`][day12] | Part [1][day12-1], [2][day12-2] | None |
```

Comment:
The new README links work:
https://github.com/petermeansrock/advent-of-code-2021/tree/day12